setup.py

```diff
@@ -67,6 +67,7 @@
         'django-wkhtmltopdf>=2.0.3,<2.1',
         'djangorestframework-xml>=1.3.0',
         'djangorestframework>=3.11.0',
+        'dotmap>=1.3.8',
```

Comment:
This is only needed for tests, so can safely be moved into the `test` part of `extras_require` below.